site/content/blog/multi-env-deployment-ytt-kapp.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Multi environment deployment with ytt and kapp"
```

Comment:
Following are the options I can think of for title - 
1. Using Ytt and kapp to Deploy a Kubernetes Application to Multiple Environments
2. Handling mutiple environments with ytt and kapp
3. Managing Multi-environments  configuration with ytt and deploying with kapp

Feel free to suggest any other title that can be a better fit.

Reply:
I really like this one:

> Handle multiple environments with ytt and kapp

It's user-oriented: it puts first what problem do _they_ have that these tools solve. It's also plain-spoken — as someone managing Kubernetes manifests, I _know_ what all these words mean.